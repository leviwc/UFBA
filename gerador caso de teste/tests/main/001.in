24 237
11 21
4 14
5 18
19 5
1 4
17 10
10 23
10 10
11 20
11 19
3 14
8 14
13 20
10 6
10 9
7 10
14 24
24 20
20 3
21 17
23 5
8 17
10 4
7 24
11 18
3 3
10 14
1 5
8 18
10 15
20 8
3 17
8 6
21 3
18 7
10 21
19 9
4 2
16 7
22 9
6 16
9 12
17 17
6 13
6 22
3 12
12 1
23 7
13 4
15 24
4 23
15 14
24 10
20 16
11 10
8 5
19 6
22 14
9 11
20 17
2 5
12 18
17 7
2 22
7 20
4 16
8 1
23 23
24 17
11 22
8 4
24 2
17 22
17 23
17 15
6 21
13 16
13 24
20 2
11 12
21 22
19 1
9 7
8 9
16 21
9 20
9 16
11 2
1 14
15 13
21 18
2 13
5 22
14 13
14 11
24 16
4 5
14 5
15 3
13 12
8 7
17 6
24 21
10 22
22 20
21 15
20 12
24 9
4 24
7 13
23 11
19 13
17 1
3 22
3 10
19 24
6 3
7 5
2 16
23 3
3 9
7 4
23 20
11 7
24 23
18 4
6 5
12 24
15 2
15 23
19 23
18 2
20 4
20 18
16 3
4 15
17 18
6 15
21 19
23 14
21 9
14 16
5 12
17 4
10 1
2 2
21 20
21 14
14 9
19 7
19 8
4 9
23 2
22 8
23 6
3 24
2 1
2 9
18 14
19 4
13 8
21 1
8 12
6 7
17 2
3 2
22 24
18 15
4 6
2 10
11 4
6 11
16 19
14 6
2 8
21 2
11 5
7 1
5 20
19 12
14 17
18 23
5 13
20 15
10 5
7 12
7 7
19 3
14 19
22 7
3 4
15 22
18 18
17 19
3 8
13 13
23 22
16 5
16 15
21 5
1 22
3 13
18 10
17 11
13 1
12 2
14 2
14 7
13 17
20 14
12 12
4 21
13 11
22 18
4 12
12 10
22 22
1 16
9 1
16 18
21 7
19 19
6 2
10 20
7 3
16 12
8 24
9 13
5 9
22 16
20 20
22 13
21 12
24 1
20 19
6 24
13 23
