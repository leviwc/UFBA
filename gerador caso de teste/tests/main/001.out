Resultado com busca binaria: 3
102.355000 s
Resultado com backtracking apenas: 3
0.025000 s
Resultado testando todas as possibilidades: 3
481.628000 s
Resultado com busca binaria otimizada: 3
17.510000 s
